docs/rc.xml.all

```diff
@@ -38,6 +38,19 @@
     <topMaximize>yes</topMaximize>
   </snapping>
 
+  <!--
+    Use GoToDesktop left | right to switch workspaces.
+    Use SendToDesktop left | right to move windows.
+    See man labwc-actions for futher information.
+  -->
+  <desktops>
+    <names>
+      <name>Workspace 1</name>
+      <name>Workspace 2</name>
+      <name>Workspace 3</name>
+    </names>
+  </desktop>
```

Comment:
Small typo I noticed. Should be `</desktops>` :)

Reply:
Great pick up. Thanks

Reply:
fixed, thanks